agent/service.go

```diff
@@ -124,7 +124,7 @@ func (as *agentService) Algo(ctx context.Context, algo Algorithm) error {
 		return fmt.Errorf("error getting current directory: %v", err)
 	}
 
-	f, err := os.Create(filepath.Join(currentDir, "algorithm"))
+	f, err := os.Create(filepath.Join(currentDir, "algo"))
```

Comment:
why is this change done

Reply:
Tests fail because algorithm dir already exists in the code base